pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -2747,6 +2815,47 @@ jobs:
       docker-image-resource: ops-docker-image-gcr,
       kubernetes-repo: census-rm-kubernetes-ops-repo}
 
+- name: "CI Deploy Ops UI Tool"
```

Comment:
I'd rather stick to the name "Ops UI" without "Tool" to avoid confusing it any more with the testing "Ops Tool"